tunelinux/config/multipath.conf.generic

```diff
@@ -47,19 +47,6 @@ devices {
     device {
         vendor "3PARdata"
         product "VV"
-        path_grouping_policy "group_by_prio"
```

Comment:
Do we not have a default settings for 3par? Why are these removed

Reply:
For 3PAR we have different combination of suggested multipath configs depending on distro type.Depending on distro type some configs needs to be enabled or removed . Since there can be a lot of combination depending on distro type and version , only the common configs which will always be required for all distro types are kept in default settings. 

If the multipath.conf file is absent , default configuration will be applied first and then based on distro type the other recommended configs will be read from config.json and applied.

At present the Default section has only 2 configs i.e vendor and product type. In the next revisions when we add the section for other distro Type, we will figure out the common configs and add it in the default settings in this file.

Reply:
Thanks for the clarification. We need to make sure all the different distros have been tested.